mobile/src/components/ColumnBookList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+import { Book } from '../models/Book';
+import { BookCard } from './BookCard';
+
+type ColumnBookListProps = { books: Book[] };
+
+/**
+ * Renders a vertical list of books with 3 books per row.
+ */
+export const ColumnBookList: React.FC<ColumnBookListProps> = ({ books }) => {
+  return (
+    <FlatList
+      data={books}
+      renderItem={({ item }) => (
+        <View style={styles.bookCard}>
+          <BookCard book={item} size={120} />
```

Comment:
I think having the size always be 120 is what's causing the issue with spacing on devices with smaller screen sizes. Changing this to depend on the screen resolution may be better. However, this would require using the dimensions of the device to calculate image size since the book image comes from a url and so we have to specify the exact dimensions of the image.

Reply:
Damn. @Anshul-Birla @aksharans maybe we should use `Dimensions` in this case since we're always showing 3 books in a row. Maybe something like `width * 0.25` so it's a little less than 1/3 to give room for spacing? Hopefully that looks good on most screen sizes.

Reply:
I think that should be fine, but if we are doing size based on the dimensions of the device, there could be a chance that those images look smaller compared to the New Books for You images, so I think it's just a matter if the design is okay with not having the All Books images being larger than the other booklist images and being a set size.

Reply:
I think that's fine. Main idea here is that density independent pixels probably aren't going to work when we're not in a scrolling container.